src/main/java/com/gui/pages/desktop/RegisterPage.java

```diff
@@ -9,101 +9,116 @@
 
 public class RegisterPage extends AbstractPage {
 
-    @FindBy (xpath = "//input[contains(@id, 'AccountFrm_firstname')]")
+    @FindBy(id = "AccountFrm_firstname")
     private WebElement firstNameField;
 
-    @FindBy (id = "AccountFrm_lastname")
+    @FindBy(id = "AccountFrm_lastname")
     private WebElement lastNameField;
 
-    @FindBy (id ="AccountFrm_email")
+    @FindBy(id = "AccountFrm_email")
     private WebElement emailField;
 
-    @FindBy (id ="AccountFrm_address_1")
+    @FindBy(id = "AccountFrm_address_1")
     private WebElement adress1Field;
 
-    @FindBy (id ="AccountFrm_city")
+    @FindBy(id = "AccountFrm_city")
     private WebElement cityField;
 
-    @FindBy (id ="AccountFrm_zone_id")
+    @FindBy(id = "AccountFrm_zone_id")
     private WebElement regionSelector;
 
-    @FindBy (xpath = "//select[contains(@id, 'AccountFrm_zone_id')]//option")
+    @FindBy(xpath = "//select[contains(@id, 'AccountFrm_zone_id')]//option")
     private List<WebElement> regionOption;
 
-    @FindBy (id ="AccountFrm_postcode")
+    @FindBy(id = "AccountFrm_postcode")
     private WebElement zipCodeField;
 
-    @FindBy (id ="AccountFrm_country_id")
+    @FindBy(id = "AccountFrm_country_id")
     private WebElement countrySelector;
 
-    @FindBy (id ="AccountFrm_loginname")
+    @FindBy(id = "AccountFrm_loginname")
     private WebElement loginNameField;
 
-    @FindBy (id ="AccountFrm_password")
+    @FindBy(id = "AccountFrm_password")
     private WebElement passwordField;
 
-    @FindBy (id ="AccountFrm_confirm")
+    @FindBy(id = "AccountFrm_confirm")
     private WebElement passwordConfirmField;
 
-    @FindBy (id ="AccountFrm_agree")
+    @FindBy(id = "AccountFrm_agree")
     private WebElement privacyPolicyCheckbox;
 
-    @FindBy (xpath = "//button[contains(@title, 'Continue')]")
+    @FindBy(xpath = "//button[contains(@title, 'Continue')]")
     private WebElement registerButton;
 
-    @FindBy (xpath = "//span[contains(text(), ' Your Account Has Been Created!')]")
+    @FindBy(xpath = "//div[@class='login-form']//b[text()='%s']")
     private WebElement accountCreatedText;
 
-    public void enterFirstName(String firstName){
-        super.sendKeysElement(firstNameField,firstName, "firstNameField");
+    public void fillOutRegistrationForm(String firstname, String lastName, String email, String adress, String city, String zipCode, String login, String password) {
```

Comment:
a lot of arguments let's map it to the pojo object
example
public void completeSignUpForm(UserService user) {